lib/CreateStudent.dart

```diff
@@ -1,135 +1,1160 @@
+                  ),
+                  enabledBorder: OutlineInputBorder(
+                    borderSide: BorderSide(color: Colors.black, width: 1.0),
+                  ),
+                  labelText: "Taluk",
+                ),
+                items: taluk.map<DropdownMenuItem<String>>((e) {
+                  return (DropdownMenuItem(
+                    child: Text(e.toString()),
+                    value: e,
+                  ));
+                }).toList(),
+                value: createReq ? talukValue : pTalukValue,
+                onChanged: (value) {
+                  setState(() {
+                    if (createReq)
+                      talukValue = value.toString();
+                    else
+                      pTalukValue = value.toString();
+                    print(talukValue);
+                  });
+                },
+              )),
+          Padding(
+              padding:
+                  EdgeInsets.only(bottom: 6.0, left: 60, right: 60, top: 10),
+              child: TextFormField(
+                decoration: new InputDecoration(
+                  isDense: true,
+                  contentPadding: new EdgeInsets.symmetric(
+                      vertical: 10.0, horizontal: 10.0),
+                  border: OutlineInputBorder(
+                      borderRadius: BorderRadius.circular(32.0)),
+                  focusedBorder: OutlineInputBorder(
+                    borderSide: BorderSide(color: Colors.black, width: 1.0),
+                  ),
+                  enabledBorder: OutlineInputBorder(
+                    borderSide: BorderSide(color: Colors.black, width: 1.0),
+                  ),
+                  hintText: 'PinCode',
+                  labelText: 'PinCode',
+                ),
+                controller: createReq ? zipCodeController : pZipCodeController,
+                validator: (val) {
+                  if (val == null || val.isEmpty) {
+                    return 'Please enter pincode';
+                  }
+                },
+              )),
+        ],
+      ),
+    )));
+  }
+
+  populatePermanentAddress() {
+    pAddressLine1Controller.value =
+        TextEditingValue(text: addressLine1Controller.text);
+    pAddressLine2Controller.value =
+        TextEditingValue(text: addressLine2Controller.text);
+    pZipCodeController.value = TextEditingValue(text: zipCodeController.text);
+    pCountryValue = countryValue;
+    pStateValue = stateValue;
+    pDistrictValue = districtValue;
+    pTalukValue = talukValue;
+  }
+  void pickDate(BuildContext context) async {
+    final initialDate = DateTime.now();
+    final newDate = await showDatePicker
+      (context: context,
+      initialDate: date,
+      firstDate: DateTime(initialDate.year),
+      lastDate: DateTime(initialDate.year+5),
+    );
+    if(newDate == null)
+      return;
+    setState(() {
+      date = newDate;
+      dateController.value=TextEditingValue(text: getDate());
+    });
   }
 }
+
+
+
+
+
```

Comment:
remove picker.dart file as it is not used anywhere

Reply:
ok